components/org.wso2.carbon.stream.processor.common/src/main/java/org/wso2/carbon/stream/processor/common/utils/config/ClusterConfiguration.java

```diff
@@ -0,0 +1,40 @@
+package org.wso2.carbon.stream.processor.common.utils.config;
```

Comment:
Move this class form jobmanager.core to stream.processor.common module. DO not duplicate code